examples/dt/neutron/CRD_files/neutron_ovn.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+    spec:
+      neutron:
+        template:
+        customServiceConfig: |
```

Comment:
some of the config is duplicated with default, and some will not work with the nic-configs, so below is the adjusted one:-
      customServiceConfig: |
        [DEFAULT]
          vlan_transparent = true
          service_plugins = qos,ovn-router,trunk,segments,log
          agent_down_time = 600
          router_distributed = true
          router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
          debug = true
          [agent]
          report_interval = 300
          [database]
          max_retries = -1
          db_max_retries = -1
          [keystone_authtoken]
          region_name = regionOne
          memcache_use_advanced_pool = True
          auth_type = password
          [nova]
          region_name = regionOne
          endpoint_type = internal
          [oslo_messaging_notifications]
          driver = noop
          [oslo_middleware]
          enable_proxy_headers_parsing = true
          [oslo_policy]
          policy_file = /etc/neutron/policy.yaml
          [placement]
          region_name = regionOne
          [ovs]
          igmp_snooping_enable = true
          [ovn]
          ovn_emit_need_to_frag = true
          [ml2]
          type_drivers = geneve,vlan,flat
          tenant_network_types = vlan,flat
          [ml2_type_vlan]
          network_vlan_ranges = datacentre:1000:200